src/Identity/ApiAuthorization.IdentityServer/test/Microsoft.AspNetCore.ApiAuthorization.IdentityServer.Tests.csproj

```diff
@@ -18,16 +18,16 @@
 
   <ItemGroup>
     <None Update="current.pfx">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Could you switch to attributes while changing this file❔ Each `Update` would fit on one line then. May even be able to fit all for updates on a single line 😺 

Might want to remove `CopyToOutputDirectory="{blah}"` entirely here too. @rainersigwald is that the default for `@(Content)` items❔

Reply:
These aren't `Content`, they are `None`, so seems like we couldn't remove these?

Reply:
Make them single line though